### PR TITLE
organized our JS code a bit. Fixed Organizations HTML layout to match…

### PR DESCRIPTION
**Pull request by GonzalesMatthew**
… wireframe. Updated Organizations card builder to match wireframe.

## Description
- organized team JS a bit: made clearer comments to separate section where we create our arrays
- cleaned up Organizations.html to match wireframe, along with styling
- in JS, improved my organizations card builder to match wireframe

## Related Issue
#49

## Motivation and Context
Match organizations.html to wireframe

## How Can This Be Tested?
Compare organizations.html to wireframe
https://www.figma.com/file/V64DhnLjKDpvmE4kjt6tEs/GitSub-Copy?node-id=5%3A20

## Screenshots (if appropriate):
Screen Shot 2021-02-13 at 12 29 32 AM
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -100,7 +101,7 @@ const repos = [
     description: 'Use the form below to create repositories of your own.',
   },
 ];
-
+// Pines Array
```
Comment (HunterJuneau) [inline main.js:104]: Typo here!

Comment (HunterJuneau) [inline main.js:104]: Typo here!

**Review by HunterJuneau — CHANGES_REQUESTED** [review]
There's a typo in the "Pins Array" comment.

**Review by hmparsons-sde — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (GonzalesMatthew) [reply main.js]: Fixed!

**Review by HunterJuneau — APPROVED** [review]

**Review by Gabrielle-Tobermann — APPROVED** [review]

**Pull request merged**